null_safety_examples/iterables/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: ../../examples/analysis_options.yaml
```

Comment:
nit: newline needed.

I'd actually recommend copying the file at `/examples/analysis_options.yaml` to `null_safety_examples/analysis_options.yaml`. It's possible (even likely) that they'll diverge at some point, and I'd also like to avoid inter-dependencies between the two sets of examples.

Reply:
makes sense, thanks! I will do that